mnist_lenet/__init__.py

```diff
@@ -0,0 +1,181 @@
+import logging
+from argparse import ArgumentParser
+import numpy
+
+from theano import tensor
+
+from blocks.algorithms import GradientDescent, Scale
+from blocks.bricks import MLP, Rectifier, Initializable, FeedforwardSequence, Softmax
+from blocks.bricks.conv import (
+    ConvolutionalLayer, ConvolutionalSequence, Flattener)
+from blocks.bricks.cost import CategoricalCrossEntropy, MisclassificationRate
+from blocks.initialization import IsotropicGaussian, Constant
+from fuel.streams import DataStream
+from fuel.datasets import MNIST
+from fuel.schemes import SequentialScheme
+from blocks.filter import VariableFilter
+from blocks.graph import ComputationGraph, apply_dropout
+from blocks.model import Model
+from blocks.monitoring import aggregation
+from blocks.extensions import FinishAfter, Timing, Printing
+from blocks.extensions.saveload import Checkpoint
+from blocks.extensions.monitoring import (DataStreamMonitoring,
+                                          TrainingDataMonitoring)
+from blocks.extensions.plot import Plot
```

Comment:
Sorry, I just moved the `Plot` extension to `blocks.extras`, but that's not being installed yet on Travis. Maybe it's easiest to get rid of the plotting for now. I'll add it back in a bit.